src/Domain/FunctionsOfTime/ReadSpecThirdOrderPiecewisePolynomial.hpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
```

Comment:
I need some time to figure out how to do this, because my git experience isn't enough for it to be obvious. I went ahead and pushed my other fixups, so I can see if the tests pass and so you can see how I responded to your other requested changes. I'll give this factoring a try next chance I have time

Reply:
I refactored my commits as best I could without spending too much time on it